### ISSUE TITLE
force_unwrapping: False positive on IUO closure property declaration

### ISSUE THREAD

**Issue by ldiqual** [realm/SwiftLint#614]
```
private var myProperty: (Void -> Void)! // Line 30
```

triggers

```
/path/code.swift:30:50: warning: Force Unwrapping Violation: Force unwrapping should be avoided. (force_unwrapping)
```

**Comment by pwc3** [comment]
I ran into the same thing. Adding a typealias for the block type works around the issue, but in this case, I just added a `// swiftlint:disable:this force_unwrapping` comment.
